pipes/WDL/workflows/tasks/taxon_filter.wdl

```diff
@@ -7,6 +7,8 @@ task deplete_taxa {
   Array[File]? bmtaggerDbs  # .tar.gz, .tgz, .tar.bz2, .tar.lz4, .fasta, or .fasta.gz
   Array[File]? blastDbs  # .tar.gz, .tgz, .tar.bz2, .tar.lz4, .fasta, or .fasta.gz
   Int?         query_chunk_size
+  Boolean?     clear_tags = false
+  String? tags_to_clear_space_separated
```

Comment:
`Array[String]?` perhaps? Then further below in the task command section you can use the `${sep=' ' tags_to_clear}` WDL string join notation.

Reply:
Also, because of the behavior of this pair of parameters, maybe it would be clearer to the user of this workflow if `tags_to_clear` were actually set to a default list that matches the argparse default in python, otherwise it's not clear that leaving it unspecified actually implies a decent default list (or why this requires two parameters instead of just one).

Reply:
I initially had it as `Array[String]`, but DNAnexus rendered a simple text box in the optional settings, and the expected input format was not clear (comma-separated, or space-separated?). Setting a default makes sense though, thanks!

Reply:
Ok, sounds like something to ask the dxWDL folks about later on.